content/reference/user/artifact_gen/index.md

```diff
@@ -258,30 +260,30 @@ If you want data to be unique cluster-wide, you have to explicitly tell Gatling
 val feeder = csv("foo.csv").shard
 ```
 
-Assuming a CSV file contains 1000 entries, and 3 Gatling nodes, the entries will be distributed the following way:
+Assuming the CSV file contains 1000 entries, and you run your simulation on 3 Gatling nodes, the entries will be distributed as follows:
 
 - First node will access the first 333 entries
 - Second node will access the next 333 entries
 - Third node will access the last 334 entries
 
 {{< alert tip >}}
-`shard` is available in Gatling OSS DSL but is a noop there. It's only effective when running tests with FrontLine.
+`shard` is available in Gatling OSS DSL but is a noop there. It's only effective when running tests with Gatling Enterprise.
 {{< /alert >}}
 
 ## Resolving Injector Location in Simulation
 
-When running a distributed test from multiple locations, you could be interested in knowing where a given injector is deployed in order to trigger specific behaviors depending on location.
+When running a distributed test from multiple locations, you could be interested in knowing where a given injector is deployed in order to trigger specific behaviors depending on the location.
 
-For example, you might want to hit `https://mydomain.co.uk` `baseUrl` if injector is deployed on AWS London, and `https://mydomain.com` otherwise.
+For example, you might want to hit `https://example.co.uk` if the injector is deployed in the `Europe - Paris` AWS region, and `https://example.com` otherwise.
```

Comment:
```suggestion
For example, you might want to hit `https://example.co.fr` if the injector is deployed in the `Europe - Paris` AWS region, and `https://example.com` otherwise.
```